conf/CMakeLists.txt

```diff
@@ -47,3 +49,20 @@ install(
   COMPONENT
     storage
   )
+
+else()
+
+install(
+  FILES
+    nebula-standalone.conf.default
+  PERMISSIONS
+    OWNER_READ
+    GROUP_READ
+    WORLD_READ
+  DESTINATION
+    etc
+  COMPONENT
+    graph
```

Comment:
`graph` may cause confusion. How to distinguish between `graph` ?

Reply:
I'll change to standalone if there's no other dependency. 
And for now graph and standalone only one will be compiled.